charts/incubator/passky-server/values.yaml

```diff
@@ -0,0 +1,103 @@
+image:
+  repository: rabbitcompany/passky-server
+  pullPolicy: IfNotPresent
+  tag: 8.1.6@sha256:ef803c3ffbabde6b79b996f5b33fcab0515f6439ccc961129d5da044c5518979
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsGroup: 0
+    runAsUser: 0
+service:
+  main:
+    ports:
+      main:
+        port: 18080
+        targetPort: 80
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            SERVER_LOCATION: "US"
+            SERVER_CORES: 1
+            ADMIN_USERNAME: "admin"
+            ADMIN_PASSWORD: ""
+            CF_TURNSTILE_SITE_KEY: "1x00000000000000000000AA"
+            CF_TURNSTILE_SECRET_KEY: "1x0000000000000000000000000000000AA"
+            DATABASE_ENGINE: mysql
+            MYSQL_HOST:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "mariadbcreds" }}'
+                key: plainhost
+            MYSQL_PORT: 3306
+            MYSQL_DATABASE: "{{ .Values.mariadb.mariadbDatabase }}"
+            MYSQL_USER: "{{ .Values.mariadb.mariadbUsername }}"
+            MYSQL_PASSWORD:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "mariadbcreds" }}'
+                key: mariadb-password
+            MYSQL_SSL: "false"
+            MYSQL_SSL_CA: "/etc/ssl/certs/ca-certificates.crt"
+            REDIS_HOST:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "rediscreds" }}'
+                key: plainhost
+            REDIS_PORT: 6379
+            REDIS_PASSWORD:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "rediscreds" }}'
+                key: redis-password
+            REDIS_LOCAL_HOST:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "rediscreds" }}'
+                key: plainhost
+            REDIS_LOCAL_PORT: 6379
+            REDIS_LOCAL_PASSWORD:
+              secretKeyRef:
+                expandObjectName: false
+                name: '{{ printf "%s-%s" .Release.Name "rediscreds" }}'
+                key: redis-password
+            MYSQL_CACHE_MODE: 0
+            MAIL_ENABLED: "false"
+            ACCOUNT_MAX: 100
+            ACCOUNT_MAX_PASSWORDS: 1000
+            ACCOUNT_PREMIUM: -1
+            YUBI_CLOUD: "https://api.yubico.com/wsapi/2.0/verify"
+            YUBI_ID: 67857
```

Comment:
Is this ID same for everyone?

Reply:
You're correct. The specific ID each user uses doesn't actually matter. However, this particular ID is linked to our company email address. In the event that the 2FA in the Passky Server is incorrectly implemented, Yubico will be able to contact the user associated with this ID.

Therefore, this ID should be unique to each application or software.